### ISSUE TITLE
[BUG][VCC-WEB] Fails to launch when the current directory is not where the executable exists

### ISSUE THREAD

**Issue by naqtn** [vrchat-community/creator-companion#54]
**Describe the bug**
VCC fails to launch when the current directory is not where the executable exists.

**To Reproduce**
1. Open the command prompt.
2. Set the current directory to where the executable file exists.
3. Launch. Then the window comes up. Close it.
4. Change the current directory to another place.
5. Launch. This time, the window doesn't come up.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version vcc_2.0.0-beta.1

**Additional context**
I get the following log when launching from the Git BASH console ( https://gitforwindows.org/ )
```
Unhandled exception. System.ArgumentException: Startup Parameters Are Not Valid:
WindowIconFile: WebApp\Dist\favicon.ico cannot be found

   at PhotinoNET.PhotinoWindow.WaitForClose()
   at VCCApp.Program.Main(String[] args)
```

**Comment by orels1** [comment]
Little context for future work: the pathing resolution for favicon is done based on the current working directory it seems

**Comment by orels1** [comment]
This should be addressed in the VCC 2.0!
[Grab it from here](https://vrchat.com/download/vcc)
